src/main/kotlin/no/nav/familie/ba/soknad/api/kontrakt/Kvittering.kt

```diff
@@ -0,0 +1,5 @@
+package no.nav.familie.ba.soknad.api.kontrakt
+
+import java.time.LocalDateTime
+
+data class Kvittering(val tekst: String, val mottattDato: LocalDateTime)
```

Comment:
Flytt alt med søknad til søknad mappe istedet for kontrakt slik som dere har gjort i ba-mottak.